cloudslang-compiler/src/test/resources/flow_with_argument_modifiers.yaml

```diff
@@ -20,9 +20,7 @@ flow:
         do:
           ops.check_Weather:
             - city: city_name
-            - country:
-                default: "'Neverland'"
-                overridable: false
+            - country: 'Neverland'
```

Comment:
It seems to me that this flow checks now the same use case as `flow_with_data`, no?